src/Goerli-DssSpell.sol

```diff
@@ -41,29 +45,98 @@ contract DssSpellAction is DssAction, DssSpellCollateralOnboardingAction {
+    address constant MCD_CURE  = 0xFA5d993DdA243A57eefbbF86Cb3a1c817Dfc7e4E;
+    address constant MCD_FLASH = 0xAa5F7d5b29Fa366BB04F6E4c39ACF569d5214075;
+
+    address immutable MCD_VAT          = DssExecLib.vat();
+    address immutable MCD_VOW          = DssExecLib.vow();
+    address immutable MCD_CAT          = DssExecLib.cat();
+    address immutable MCD_DOG          = DssExecLib.dog();
+    address immutable MCD_SPOT         = DssExecLib.spotter();
+    address immutable MCD_POT          = DssExecLib.pot();
+    address immutable MCD_ESM          = DssExecLib.esm();
+    address immutable ILK_REGISTRY     = DssExecLib.reg();
+    address immutable MCD_END_OLD      = DssExecLib.end();
+    address immutable MCD_FLASH_LEGACY = DssExecLib.getChangelogAddress("MCD_FLASH");
+
     function actions() public override {
         // ---------------------------------------------------------------------
         // Includes changes from the DssSpellCollateralOnboardingAction
-        onboardNewCollaterals();
+        // onboardNewCollaterals();
+
+        // -------------------------- Cure + New End ---------------------------
+        DssExecLib.setValue(MCD_END, "wait", EndAbstract(MCD_END_OLD).wait());
+        DssExecLib.setContract(MCD_END, "cure", MCD_CURE);
+        DssExecLib.authorize(MCD_CURE, MCD_END);
+
+        require(MCD_VAT == EndAbstract(MCD_END_OLD).vat(), "sanity-error-vat");
+        DssExecLib.setContract(MCD_END, "vat", MCD_VAT);
+        DssExecLib.authorize(MCD_VAT, MCD_END);
+        DssExecLib.deauthorize(MCD_VAT, MCD_END_OLD);
+
+        require(MCD_VOW == EndAbstract(MCD_END_OLD).vow(), "sanity-error-vow");
+        DssExecLib.setContract(MCD_END, "vow", MCD_VOW);
+        DssExecLib.authorize(MCD_VOW, MCD_END);
+        DssExecLib.deauthorize(MCD_VOW, MCD_END_OLD);
+
+        require(MCD_CAT == EndAbstract(MCD_END_OLD).cat(), "sanity-error-cat");
+        DssExecLib.setContract(MCD_END, "cat", MCD_CAT);
+        DssExecLib.authorize(MCD_CAT, MCD_END);
+        DssExecLib.deauthorize(MCD_CAT, MCD_END_OLD);
+
+        require(MCD_DOG == EndAbstract(MCD_END_OLD).dog(), "sanity-error-dog");
+        DssExecLib.setContract(MCD_END, "dog", MCD_DOG);
+        DssExecLib.authorize(MCD_DOG, MCD_END);
+        DssExecLib.deauthorize(MCD_DOG, MCD_END_OLD);
+
+        require(MCD_SPOT == EndAbstract(MCD_END_OLD).spot(), "sanity-error-spot");
+        DssExecLib.setContract(MCD_END, "spot", MCD_SPOT);
+        DssExecLib.authorize(MCD_SPOT, MCD_END);
+        DssExecLib.deauthorize(MCD_SPOT, MCD_END_OLD);
+
+        require(MCD_POT == EndAbstract(MCD_END_OLD).pot(), "sanity-error-pot");
+        DssExecLib.setContract(MCD_END, "pot", MCD_POT);
+        DssExecLib.authorize(MCD_POT, MCD_END);
+        DssExecLib.deauthorize(MCD_POT, MCD_END_OLD);
+
+        DssExecLib.setContract(MCD_ESM, "end", MCD_END);
+        DssExecLib.authorize(MCD_END, MCD_ESM);
+        DssExecLib.deauthorize(MCD_END_OLD, MCD_ESM);
+
```

Comment:
Should we deauthorize pause proxy as well from the old End? just to make the removal cleaner.

Reply:
In general we do not do it, we do not remove pause proxy from old contracts.